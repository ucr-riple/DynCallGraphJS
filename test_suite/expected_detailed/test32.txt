{
    'global (test32.js:9:1:11:3)': [
        'unshift (Native)(test32.js:9:1:11:3)'
    ],
    'global (test32.js:12:1:12:21)': [
        'anon (test32.js:9:25:11:2)'
    ]
}