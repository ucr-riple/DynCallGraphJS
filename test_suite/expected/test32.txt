{ '(test32.js:9:1:11:3)': [ 'unshift (Native)' ],
  '(test32.js:12:1:12:21)':
   [ '(test32.js:9:25:11:2)' ] }